template_info.json

```diff
@@ -0,0 +1,19 @@
+{
+   "name":"Q&A Chat Generation App",
+   "description":"Q&A Chat Generation App",
+   "templateMetadata": {
+      "tags": [
+         "Streamlit",
+         "ChatGPT",
+         "GenerativeAI"
+      ],
+      "source": {
+         "repositoryUrl": "https://github.com/datarobot-oss/qa-app-streamlit",
+         "releaseTag": "10.2"
+      },
+      "previewImage": "qa-app-preview.png"
+   },
+   "templateType":"customApplicationTemplate",
+   "templateSubType":"customApplicationTemplate",
+   "executionEnvironmentName":"[Experimental] Python 3.9 Streamlit"
```

Comment:
Quick question here - Do we use name to match up the template with env? 
 I added the 'release ready' env to move away from Experimental but it hasnt been installed on staging/prod yet, should we stick to old first or go with new one from the start? "[DataRobot] Python 3.12 Streamlit"
 https://github.com/datarobot/datarobot-user-models/blob/master/public_dropin_environments/python312_streamlit/env_info.json

Reply:
Hm, for this case we need also update name here also: https://github.com/datarobot/datarobot-custom-templates/blob/main/execution_environments/python39_streamlit/env_info.json

We can update name here, because we still do not have jobs for pushing data to DR. But before starting those jobs, we need to sync naming everywhere.

Reply:
Ben suggested we leave the old 3.9 Experimental as it is and then add 3.12 as brand new env